test/views/guidance/StartViewSpec.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.guidance
+
+import scala.collection.JavaConverters.asScalaIteratorConverter
+
+import base.Injector
+import tools.Stubs
+import views.declaration.spec.UnitViewSpec
+import views.html.guidance.start
+import views.tags.ViewTest
+
+@ViewTest
+class StartViewSpec extends UnitViewSpec with Stubs with Injector {
+
+  private val startPage = instanceOf[start]
+  private val view = startPage()(request, messages)
+
+  "'Start' view" should {
+
+    "display page header" in {
+      view.getElementsByTag("h1").first must containMessage("guidance.start.title")
+    }
+  }
+}
```

Comment:
Don't know if there was any other convention agreed for these pages, but normally we tend to test in more detail.